python/cudf_polars/cudf_polars/dsl/expressions/unary.py

```diff
@@ -25,13 +25,14 @@
 class Cast(Expr):
     """Class representing a cast of an expression."""
 
-    __slots__ = ()
+    __slots__ = ("strict",)
     _non_child = ("dtype",)
 
-    def __init__(self, dtype: DataType, value: Expr) -> None:
+    def __init__(self, dtype: DataType, value: Expr, *, cast_option: int = 0) -> None:
```

Comment:
This will break `reconstruct`. Please only use positional arguments.
```suggestion
    def __init__(self, dtype: DataType, cast_option: int, value: Expr) -> None:
```